deepmd/pd/model/descriptor/se_atten.py

```
@@ -0,0 +1,1073 @@
+        attn_layer: int = 2,
+        attn_dotr: bool = True,
+        attn_mask: bool = False,
+        activation_function="tanh",
+        precision: str = "float64",
+        resnet_dt: bool = False,
+        scaling_factor=1.0,
+        normalize=True,
+        temperature=None,
+        smooth: bool = True,
+        type_one_side: bool = False,
+        exclude_types: list[tuple[int, int]] = [],
```

Ruff (0.8.0)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)